backend/src/test/java/com/bakdata/conquery/integration/json/FormTest.java

```diff
@@ -81,18 +82,18 @@
 	@Override
 	public void importRequiredData(StandaloneSupport support) throws Exception {
 
-		LoadingUtil.importTables(support, content);
+		IntegrationUtils.importTables(support, content);
 		support.waitUntilWorkDone();
 		log.info("{} IMPORT TABLES", getLabel());
 
 		importConcepts(support);
 		support.waitUntilWorkDone();
 		log.info("{} IMPORT CONCEPTS", getLabel());
 
-		LoadingUtil.importTableContents(support, content);
+		IntegrationUtils.importTableContents(support, Arrays.asList(content.getTables()), support.getDataset());
```

Comment:
Mache ambesten aus dem `table` Array in RequiredData gleich eine Liste, dann musst du hier nichts wrappen